Tasks/Common/webdeployment-common/variableutility.ts

```diff
@@ -0,0 +1,10 @@
+
+export function isPredefinedVariable(variable: string): boolean {
```

Comment:
just moving all the utilities related to variable substitution. It becomes more difficult to mock the entire utility function for L0 tests.